frontend/src/metabase/ui/theme.ts

```diff
@@ -95,6 +95,7 @@ export const getThemeOverrides = (): MantineThemeOverride => ({
       },
     },
   },
+  fontFamily: "var(--mb-default-font-family), Lato, sans-serif",
```

Comment:
I think we should follow our root style for this. I've not seen we specify Lato after the default font before.

https://github.com/metabase/metabase/blob/256835b0c0773509bce880831273a07ecde694fd/frontend/src/metabase/css/core/base.styled.ts#L5


```suggestion
  fontFamily: "var(--mb-default-font-family), sans-serif",
```